admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -293,6 +279,84 @@ On the one hand the memory usage is reduced, on the other hand there is more net
 We recommend using the option with a value high enough to process a large number of notifications, but not so large to crash the process.
 Between 200 and 500 should be fine, and we'll likely process all the notifications in one go.
 
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are several ways to supply a password:
+
+#. Interactively in response to a password prompt.
+
+   ::
+
+      sudo -u www-data ./occ wnd:listen <host> <share> <username>
+
+#. Sent as a parameter to the command.
+
+   ::
+
+      sudo -u www-data ./occ wnd:listen <host> <share> <username> <password>
+
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. Note that the password must be in plain text inside the file, and neither spaces nor newline characters will be removed from the file by default, unless the ``--pasword-trim`` option is added. The password file must be readable by the apache user (or www-data)
+
+   ::
+
+      sudo -u www-data ./occ wnd:listen <host> <share> <username> \
+        --password-file=/my/secret/password/file
+
+   ::
+
+      sudo -u www-data ./occ wnd:listen <host> <share> <username> \
+        --password-file=/my/secret/password/file --password-trim
+
+   .. note::
+      If you use the ``--password-file`` switch, the entire contents of the file will be used for the password, so please be careful with newlines.
+
+   .. warning::
+      If using ``--password-file`` make sure that the file is only readable by the
+      apache / www-data user and inaccessible from the web, in order to prevent
```

Comment:
s/, in order to prevent tampering or leaking of the information./. This prevents tampering or leaking of the information./g